packages/openneuro-app/src/scripts/utils/gtag.js

```diff
@@ -0,0 +1,3 @@
+export const pageview = (...args) => window && window.gtag(...args)
```

Comment:
The call for [page_view now looks like this](https://developers.google.com/analytics/devguides/collection/gtagjs/pages#page_view_event):
```ts
gtag('event', 'page_view', {
  page_title: '<Page Title>',
  page_location: '<Page Location>',
  page_path: '<Page Path>'
  send_to: '<GA_MEASUREMENT_ID>'
})```